src/app/objects/resume.ts

```diff
@@ -0,0 +1,55 @@
+import { template } from './template';
+
+export interface resume {
+  id: string;
+  template: template;
+  header: header;
+  workExps: workExp[];
+  educations: education[];
+  certificates?: string[];
+  skills: string[];
+  summary: string;
+}
+
+export interface header {
+  name: string;
+  profession: string;
+  email: string;
+  phone: string;
+  address: {
```

Comment:
This is more a question rather than a comment. I saw a few fields including address and time that share the same name but the fields in them are different. How does the serialization and deserialization work?